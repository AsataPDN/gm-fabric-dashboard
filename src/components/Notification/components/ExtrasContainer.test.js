import React from "react";
import { shallow } from "enzyme";
import ExtrasContainer from "./ExtrasContainer";

describe("ExtrasContainer", () => {
  it("matches snapshot", () => {
    const aExtrasContainer = shallow(<ExtrasContainer />);
    expect(aExtrasContainer).toMatchSnapshot();
  });
});
