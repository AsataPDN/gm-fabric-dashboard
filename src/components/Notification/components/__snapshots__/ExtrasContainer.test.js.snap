// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExtrasContainer matches snapshot 1`] = `
.c0 {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 1;
  pointer-events: none;
}

.c0 > * {
  pointer-events: auto;
}

<div
  className="c0"
/>
`;
